server/concepts/Slack.ts

```diff
@@ -0,0 +1,12 @@
+import { WebClient } from '@slack/web-api';
+
+const token = process.env.SLACK_APP_TOKEN;
+
+const slackConnection = new WebClient(token);
```

Comment:
Shouldn't `slackConnection` be a property of class Slack?

Reply:
I thought about it. I'm not sure. this is a static class hmm Hmm okej `Slack.connection` sound good

Reply:
@eastemployeer please make review after changes